saltstack/base/salt/luks/bin/populate-luks-volume.sh

```diff
@@ -157,12 +157,23 @@ process_freeipa_locations() {
   fi
 }
 
+process_cm_locations() {
+  if [[ "$IS_FREEIPA" != "true" ]]
```

Comment:
This certainly works. On the other hand, I think `/etc/cloudera-scm-server` can only be found on `"$IS_GATEWAY" == "true"` nodes. (Actually, on exactly one gateway node I think, the primary one, because CM Server is not deployed in a HA fashion yet.) The `cloudera-scm-agent` locations are indeed for every node, though.

Reply:
I checked, and on the `idbroker` instance of the DL for example, the `etc/cloudera-scm-server` directory does exist, and contains the following:
```console
[root@shorvath-aws-dl-idbroker0 cloudera-scm-server]# ls -la
total 40
drwxr-xr-x.   2 cloudera-scm cloudera-scm   98 Aug 27 12:43 .
drwxr-xr-x. 121 root         root         8192 Sep  3 10:30 ..
-rw-r--r--.   1 root         root         8197 Aug 22 00:59 cmSubCaCert.profile
-rw-------.   1 cloudera-scm cloudera-scm  757 Aug 22 00:59 db.properties
-rw-r--r--.   1 root         root         4495 Aug 22 00:59 log4j.properties
-rw-r--r--.   1 root         root           80 Aug 27 12:43 repo_url.txt
```
Of these, the `db.properties` file could contain the CM DB password for example. Although it seems like this file only contains actual values on the gateway instance. Still, it might make sense to protect these files.

Reply:
This `28K` in total btw.

Reply:
Oh, I see. Then let us leave it as is. :smile: